docs/modules/ROOT/attachments/tekton/camel-k-pipeline-task-definition.yaml

```diff
@@ -35,7 +35,7 @@ metadata:
 spec:
   steps:
     - name: install
-      image: apache/camel-k:1.0.0-M2
+      image: docker.io/apache/camel-k:1.9.0-SNAPSHOT
```

Comment:
I think it should go against the latest stable `1.8.0` instead. That image was not released and will end up with pulling errors.

Reply:
I have modified to stable version 1.8.0